pkg/notifications/shoutrrr_test.go

```diff
@@ -51,6 +51,30 @@ func TestShoutrrrTemplate(t *testing.T) {
 	require.Equal(t, "info: foo bar\n", s)
 }
 
+func TestShoutrrrStringFunctions(t *testing.T) {
+	cmd := new(cobra.Command)
+	flags.RegisterNotificationFlags(cmd)
+	err := cmd.ParseFlags([]string{"--notification-template={{range .}}{{.Level | ToUpper }}: {{.Message | ToLower }} {{.Message | Title }}{{println}}{{end}}"})
```

Comment:
```suggestion
	err := cmd.ParseFlags([]string{"--notification-template={{range .}}{{.Level | printf "%v" | ToUpper }}: {{.Message | ToLower }} {{.Message | Title }}{{println}}{{end}}"})
```
I think this ought to fix the failed test, but I haven't tried it.